nixos/modules/services/misc/gitolite.nix

```diff
@@ -7,6 +7,19 @@ let
   # Use writeTextDir to not leak Nix store hash into file name
   pubkeyFile = (pkgs.writeTextDir "gitolite-admin.pub" cfg.adminPubkey) + "/gitolite-admin.pub";
   hooks = lib.concatMapStrings (hook: "${hook} ") cfg.commonHooks;
+  orNull = cond: val: if cond then val else null;
+  rcAppendFile =
+    orNull (cfg.extraGitoliteRc != "") (pkgs.writeText "gitolite-rc-append" rcAppendText);
```

Comment:
IMHO, I think it'd be easier to read if using plain "if/then/else" instead of having the "orNull" function.

Reply:
Well, it would make the line too long, or require splitting the if-then-else construct. I'll see if it can be rewritten elegantly.